config/locales/en.yml

```diff
@@ -62,6 +62,11 @@ en:
       already_sent_back: This application was already sent back to the provider
       already_completed: This application was already marked as complete
       already_marked_as_ready: This application was already marked as ready for assessment
+      cannot_complete_when_sent_back: This application was already sent back to the provider
```

Comment:
This looks great much cleaner implementation. Tiny thing - do you think it might be worth nesting this under `errors.` or the module name?

Reply:
Thanks does happen later in the series :)! Thanks for the review.